server/core/store_option.go

```diff
@@ -82,6 +82,16 @@ func SetStoreState(state metapb.StoreState) StoreCreateOption {
 	}
 }
 
+// OfflineStore offline a store
+func OfflineStore(physicallyDestroyed bool) StoreCreateOption {
+	return func(store *StoreInfo) {
+		meta := proto.Clone(store.meta).(*metapb.Store)
```

Comment:
Why is `Clone` needed here? It seems that the `StoreCreateOption ` in this file does not need to clone `meta`.